tests/test_project.py

```diff
@@ -1703,7 +1703,9 @@ class A(FlowProject):
             def foo_operation(job):
                 pass
 
-    @pytest.mark.filterwarnings("ignore:*with_directives*:FutureWarning")
+    @pytest.mark.filterwarnings(
+        "ignore:.*with_directives has been deprecated.*:FutureWarning"
```

Comment:
@b-butler The message is a regex, not a glob. Previously it started with `*`, which means "repeat previous character". This caused pytest to crash because no previous character was defined for a string starting with `*`.